mava/configs/default_rec_iql.yaml

```
@@ -0,0 +1,7 @@
+defaults:
+  - _self_
+  - logger: rec_iql
+  - arch: anakin
+  - system: q_learning/rec_iql
+  - env: smax
+  - network: rnn
```

NIT: Can we keep a similar order and put env last?
```suggestion
  - network: rnn
  - env: smax
```